### DISCUSSION TITLE
Using custom object detecting model

### DISCUSSION THREAD

**Discussion by olexanje** [Kotlin/kotlindl#453] [Q&A] [answered]
Hi everyone!) I'm new to DL, and unfortunately can't find way to add and use my (trained in [_jupyter notebook_](https://www.youtube.com/watch?v=yqkISICHH-U&list=WL)) model. Or, may by, exists way to train model for object detection in kotlindl?

**Comment by zaleslaw** [comment, marked as answer]
For now, there is no supported approach to training a custom object detection model because we're providing object detection models as fully trained models in ONNXModelHub.

To train it with TensorFlow module, there are not enough layers to train the model.
